k8s/overlays/local/common-application.yml

```diff
@@ -0,0 +1,22 @@
+powsybl-ws:
+  environment: local_
```

Comment:
When we deploy the application locally using minikube, do we want to share the database with the one we use for day to day dev (docker compose one) ? So for you the answer is no?

Reply:
removed